src/faldbt/project.py

```diff
@@ -117,12 +117,12 @@ def __init__(self, project_dir: str, profiles_dir: str):
             parse.get_dbt_results(self.project_dir, self._config)
         )
 
-        normalized_source_paths = normalize_directories(
-            project_dir, self._config.source_paths
+        normalized_model_paths = normalize_directories(
+            project_dir, self._config.model_paths
```

Comment:
is this changing now and source-paths is called model-paths?

Reply:
yes